package.json

```diff
@@ -18,7 +18,6 @@
     "test": "npm run lint && npm run typescript && npm run unit",
     "tdd": "npm run unit -- --watch",
     "react:16": "enzyme-adapter-react-install 16",
-    "react:15": "enzyme-adapter-react-install 15.5",
```

Comment:
The tests are currently failing on Travis because `bin/test` still references this.

https://github.com/danreeves/react-tether/blob/c2072b8b866fe7edce8642c3dd76b6843adb406b/bin/test#L11-L14

https://github.com/danreeves/react-tether/blob/c2072b8b866fe7edce8642c3dd76b6843adb406b/bin/test#L25

Reply:
Thanks, didn't even notice that!